backend/src/main/java/ai/verta/modeldb/health/HealthServiceImpl.java

```diff
@@ -77,13 +78,27 @@ private ServingStatus getServingStatus(HealthCheckRequest request) {
       globalStatus = getStatus("");
       if (request.getService().equals("ready")) {
         if (globalStatus == ServingStatus.SERVING) {
-          setStatus("ready", modelDBHibernateUtil.checkReady());
+          ServingStatus hibernateConnectionStatus = modelDBHibernateUtil.checkReady();
```

Comment:
do we still have hibernate stuff around???

Reply:
Yes. we have it. Still we have the Repository and Dataset service impl with the hibernate implementation.